etna/transforms/math/sklearn.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         ValueError:
             if incorrect mode given
         """
+        super().__init__(required_features="all")
```

Comment:
The list of required features depends in `in_column`, read the docstring and implement the necessary logic